File: Assignment 16/Question4.java
import java.util.Scanner;

class Reverse
{
    public int iDigit = 0 ;
    public void ReverseDigit(int iNo)
    { 
        while (iNo !=0)
        {
            iDigit = iNo%10 ;
            System.out.println(iDigit);
            iNo= iNo/10;
        }
    }

}

class Question4
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);
        int iValue = 0 ;

         System.out.println("Enter your Number :");
        iValue = sobj.nextInt();

        Reverse robj = new Reverse();

        robj.ReverseDigit(iValue);

    }
}

File: Assignment 7/Question4.c
#include <stdio.h>

void PrintOddNo(int iNo)
{
    int iCnt = 0;
    for (iCnt =1 ; iCnt<=iNo ;iCnt++)
    {
        if (iCnt % 2 != 0)
        {
        printf("%d\t",iCnt);
        }
        
    }
}
int main()
{
    int iValue = 0;
    printf("Enter Number: ");
    scanf("%d",&iValue);
    
    PrintOddNo(iValue);
    return 0 ;
}
